src/main/enb.ts

```diff
@@ -47,3 +47,11 @@ export async function deleteEnbFiles(skyrimDirectory: string) {
   }
   fs.rmdirSync(`${skyrimDirectory}/enbseries`, { recursive: true });
 }
+
+export async function checkEnbFiles(skyrimDirectory: string) {
+  logger.info("Checking ENB Files");
+  const existingEnbFiles = enbFiles.filter((file) =>
+    fs.existsSync(`${skyrimDirectory}/${file}`)
+  );
+  return existingEnbFiles === enbFiles;
+}
```

Comment:
Minor tweak but might be worth clarifying what is being checked.
Also, we could make this more generic so it can check any directory for ENB files.
```suggestion
export async function checkEnbFilesExist(directory: string) {
  logger.info(`Checking which ENB files exist in ${directory}`);
  const existingEnbFiles = enbFiles.filter((file) =>
    fs.existsSync(`${directory}/${file}`)
  );
  return existingEnbFiles === enbFiles;
}
```